pkg/sql/mem_metrics.go

```diff
@@ -59,13 +59,25 @@ const log10int64times1000 = 19 * 1000
 
 // MakeMemMetrics instantiates the metric objects for an SQL endpoint.
 func MakeMemMetrics(endpoint string, histogramWindow time.Duration) MemoryMetrics {
-	prefix := "sql.mon." + endpoint
-	MetaMemMaxBytes := metric.Metadata{Name: prefix + ".max"}
-	MetaMemCurBytes := metric.Metadata{Name: prefix + ".cur"}
-	MetaMemMaxTxnBytes := metric.Metadata{Name: prefix + ".txn.max"}
-	MetaMemTxnCurBytes := metric.Metadata{Name: prefix + ".txn.cur"}
-	MetaMemMaxSessionBytes := metric.Metadata{Name: prefix + ".session.max"}
-	MetaMemSessionCurBytes := metric.Metadata{Name: prefix + ".session.cur"}
+	prefix := "sql.mem." + endpoint
```

Comment:
"max" is the high water mark of memory allocations for that monitor/category.
"current" is the currently registered allocations for that monitor/category.
There is a bunch of endpoints, they are documented in `sql/mon/mem_usage.go`.